bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -33,7 +33,7 @@ metadata:
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0
```

Comment:
The manifests were changed which should not happen. Please install SDK `operator-sdk-v1.13.0+git` and re-generate manifests.

Reply:
hi, i can not install operator-sdk @v1.13.0+git. i try to install operator-sdk from github. but the can not find "v1.13.0+git" tag to be install. Are ther any guides to install the excat version?

Reply:
Install 1.13.1